.github/CONTRIBUTING.md

```diff
@@ -1,9 +1,9 @@
 # So you want to contribute to PCSX2? Great
 
-As a first step, please review these links as they'll help you understand how the development of the PCSX2 works. These pages are all found on the github [wiki](https://github.com/PCSX2/pcsx2/wiki). You should probably also check out the [forums](http://forums.pcsx2.net/index.php).
+As a first step, please review these links as they'll help you understand how the development of PCSX2 works. These pages are all found on the GitHub [wiki](https://github.com/PCSX2/pcsx2/wiki). You should probably also check out the [forums](https://forums.pcsx2.net/index.php).
```

Comment:
@xTVaser @RedDevilus Do we really want to direct people to forums in the contribution guide?

Reply:
I don't mind personally due to good threads for people.

Reply:
Which threads? Care to give an example of something that would actually help new contributors?
Not against it but I'm not sure which usefulness new devs can get out of it

Reply:
I suppose it's more patch creators and compatability reports and some other threads i can't think of. I agree it's more a sidenote than an obvious pro to contributors.

Reply:
I'll wait for @xTVaser to give his thoughts and maybe reformulate this then

Reply:
I don't have a strong opinion (I've never been on the forums myself).  But imo, if you are contributing the forums isn't really the relevant place at all.  If the wiki has critical docs related to developing, then that should be pulled out and centralized where the development actually happens -- github.